src/stores/mints.js

```diff
@@ -55,9 +55,12 @@ export const useMintsStore = defineStore('mints', {
         this.showAddMintDialog = false;
       }
     },
-    activateMint: async function (url, verbose = false) {
-      const workers = useWorkersStore()
-      if (url === this.activeMintUrl) {
+    activateMint: async function (url, verbose = false, force = false) {
+      const workers = useWorkersStore();
+      if (url === this.activeMintUrl && !force) {
```

Comment:
This is the actual change. We need this because otherwise `this.activeProofs` will stay empty forever and the user won't be able to use the wallet when they migrate from before where `this.activeProofs` was not in the db.